.envrc

```diff
@@ -15,7 +15,14 @@ REPO_HOST_PATH=$(realpath $CURRENT_PATH/..)
 REPO_NODE_PATH="/srv/git"
 REPO_URI="http://git-http-backend/git/${REPO_NAME}"
 REPO_BRANCH=$(git branch --show-current)
-K8S_VERSION="kindest/node:v1.27.3@sha256:9dd3392d79af1b084671b05bcf65b21de476256ad1dcc853d9f3b10b4ac52dde"
+
+OS_ARCH=$(uname -m)
+if [ "$OS_ARCH" == "arm64" ] || [ "$OS_ARCH" == "aarch64" ]; then
+  K8S_VERSION="kindest/node:v1.27.3@sha256:de0b3dfe848ccf07e24f4278eaf93edb857b6231b39773f46b36a2b1a6543ae9" # arm64 image
```

Comment:
This is great @Pehesi97, can we add a comment with the URL to the images so it's easier for people to find new versions? Thanks ;)